bigquery/gcloud/aio/bigquery/bigquery.py

```diff
@@ -60,10 +60,15 @@ def __init__(self, dataset_name: str, table_name: str,
         self.token = token or Token(service_file=service_file, scopes=SCOPES,
                                     session=self.session.session)
 
+
     async def project(self) -> str:
         if self._project:
             return self._project
 
+        if BIGQUERY_EMULATOR_HOST:
+            self._project = str(os.environ.get('BIGQUERY_PROJECT_ID', 'dev'))
```

Comment:
I think this can just be `self._project = BIGQUERY_EMULATOR_HOST`, since you already have a truthiness check the line above.

Reply:
But this is for `BIGQUERY_PROJECT_ID`, I think this is fine.

Reply:
yeah, it's for project 👉👈

Reply:
:facepalm: yup, you two are correct